Cargo.toml

```diff
@@ -1,67 +1,35 @@
 [package]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
+description = "Pairing threshold cryptography"
+edition = "2021"
+license = "GPL-3.0"
 name = "blsttc"
-version = "4.0.0"
-authors = [
-  "Vladimir Komendantskiy <komendantsky@gmail.com>",
```

Comment:
maybe should keep previous authors and just add maidsafe devs.....?

Reply:
I removed the previous authors (from the threshold_crypto crate) mainly because it doesn't seem like contacting those authors would be helpful to the person doing the contacting. I can understand the desire to still give credit, but from a practical point of view, I feel they aren't suitable points of contact. Keep or remove, what's the preference here? @joshuef